### PR TITLE
#14 Added Node.js Interactive 2017 Vancouver, BC to mentions section

### PR DESCRIPTION
**Pull request by Weeper**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -48,6 +48,11 @@ var globals = {
     ],
     "mentions": [
         {
+            "date": "October 16, 2017",
```
Comment (shakeelmohamed) [inline index.js]: The actual date of the session was October 5, let's use that date

File: index.js [hunk 2]
```diff
@@ -48,6 +48,11 @@ var globals = {
...
+            "title": "Node.js Interactive 2017 Vancouver, BC",
```
Comment (shakeelmohamed) [inline index.js]: Let's include the session title, `Maintainer Burden Group Therapy - Node.js Interactive 2017 Vancouver, BC1

**Comment by Weeper** [comment]
No problem, @shakeelmohamed, made the changes.

**Comment by shakeelmohamed** [comment]
Thanks @Weeper, happy hacking 🎃

**Comment by Weeper** [comment]
You are welcome @shakeelmohamed 👍

**Pull request merged**